content.sql

```diff
@@ -0,0 +1,15 @@
+
+SELECT *
+FROM Picture;
+
+select *
```

Comment:
Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc